### PR TITLE
[FiveTV] Add new regex and fix bad test

### LINKED ISSUE #13354: SITE REQUEST: www.5-tv.ru
Dear coders, I just visited this 5-tv.ru site, and I wanted to get news video from it, got no success.
Please, add this 5-tv.ru into YDL supported sources list. Thanks in advance!
The log is:
C:\>youtube-dl -v -F http://www.5-tv.ru/programs/broadcast/509514/
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-F', 'http://www.5-tv.ru/programs/broadcast/509514/']
[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
[debug] youtube-dl version 2017.06.12
[debug] Python version 3.4.4 - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg 3.3.1, ffprobe 3.3.1, rtmpdump 2.4
[debug] Proxy map: {}
[FiveTV] 509514: Downloading webpage
ERROR: Unable to extract video url; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its completeoutput.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkyaecyzu\build\youtube_dl\YoutubeDL.py", line 762,in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkyaecyzu\build\youtube_dl\extractor\common.py", line 433, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkyaecyzu\build\youtube_dl\extractor\fivetv.py", line 74, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpkyaecyzu\build\youtube_dl\extractor\common.py", line 782, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract video url; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

### PR DESCRIPTION
**Pull request by rrooij**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

The site uses flowplayer for most videos now. There was also a bad test which always failed if a new episode got released.

Fixes #13354

### CODE REVIEW

File: youtube_dl/extractor/fivetv.py [hunk 1]
```diff
@@ -39,14 +39,6 @@ class FiveTVIE(InfoExtractor):
             'duration': 180,
         },
     }, {
-        'url': 'http://www.5-tv.ru/glavnoe/#itemDetails',
-        'info_dict': {
-            'id': 'glavnoe',
-            'ext': 'mp4',
-            'title': 'Итоги недели с 8 по 14 июня 2015 года',
```
Comment (dstftw) [inline youtube_dl/extractor/fivetv.py]: This may be matched with `re:` prefixed string.

Comment (rrooij) [inline youtube_dl/extractor/fivetv.py]: Good idea.

**Comment by rrooij** [comment]
@dstftw Fixed the test. However, the title contains weird spaces. Do we have to "sanitize" the title?

File: youtube_dl/extractor/fivetv.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ class FiveTVIE(InfoExtractor):
         'info_dict': {
             'id': 'glavnoe',
             'ext': 'mp4',
-            'title': 'Итоги недели с 8 по 14 июня 2015 года',
+            'title': r're:^Итоги недели с \d+ по \d+ июня \d{4} года$',
```
Comment (dstftw) [inline youtube_dl/extractor/fivetv.py]: That's not enough. `июня` is month name.

**Comment by rrooij** [comment]
Fixed month name

**Comment by rrooij** [comment]
Is it OK now?

**Pull request merged**